app/newRoot.css

```diff
@@ -529,3 +529,10 @@ ol {
     flex-direction: column;
   }
 } /* end mobile */
+p,
+button {
+  font-family: Poppins;
```

Comment:
this is specified in body, why does it need to be specified again?

Reply:
yes. that's why buttons still are Arial